package.json

```diff
@@ -3,11 +3,25 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "client:dev": "yarn workspace client start"
+    "client:dev": "yarn workspace react-app start",
```

Comment:
I think maybe this got changed  with a merge,
The script for client should be `"client:dev": "yarn workspace client start"`